src/main/java/org/jabref/model/ai/AiProvider.java

```diff
@@ -3,10 +3,11 @@
 import java.io.Serializable;
 
 public enum AiProvider implements Serializable {
-    OPEN_AI("OpenAI", "https://openai.com/policies/privacy-policy/", "https://openai.com/policies/privacy-policy/"),
+    OPEN_AI("OpenAI", "https://api.openai.com/v1", "https://openai.com/policies/privacy-policy/"),
```

Comment:
"https://api.openai.com/v1" leads to a "404". Is this correct?

Reply:
WHOA, I left this in the code? Thank you for fixing.

`https://api.openai.com/v1` is the right link. It just doesn't have any response on `GET https://api.openai.com/v1`, you need to use a specific endpoint. You can try out `https://api.openai.com/v1/chat/completions` in a browser, and it will return you an error that you didn't include an API key, which is right

Reply:
The complete request path for openai should be https://api.openai.com/v1/chat/completions. when I set https://api.openai.com/v1, it will get the right endpoint(https://api.openai.com/v1/chat/completions) in the end(as you can see the screenshot below). The reason should be that the chatClient() method from Openai do the path splicing work that make https://api.openai.com/v1 into https://api.openai.com/v1/chat/completions. I set https://api.openai.com/v1/chat/completions at the beginning, but I got the https://api.openai.com/v1/chat/completions/chat/completions at the end.
![image](https://github.com/user-attachments/assets/46afc7dc-f465-4b83-9feb-62573850d854)

Could you please show me a screenshot or something? Because it can work on my computer.
model: gpt-4o-mini
![image](https://github.com/user-attachments/assets/157619b7-7a81-46e5-9793-400f02ee77eb)

![image](https://github.com/user-attachments/assets/b528de93-170b-4bd2-b395-5c53c2cf3d0a)


model: gpt-4o
![image](https://github.com/user-attachments/assets/e60dbd60-ec11-4dd5-a3e4-d41b375cc347)

![image](https://github.com/user-attachments/assets/4a64a5d2-ed5f-4c50-868a-3c49f1ec1946)

model: gpt-4
![image](https://github.com/user-attachments/assets/b047ece7-68ce-4595-a031-13b0b9ce2c88)

![image](https://github.com/user-attachments/assets/5a6062eb-a034-4f96-8dab-884a94cd6e16)